[ipcamera] Add new channels for Dahua API-based doorphones (#13313)
openhab/openhab-addons: bundles/org.openhab.binding.ipcamera/README.md

| `tooDarkAlarm` | Switch (read only) | ONVIF cameras only will reflect the status of the ONVIF event of the same name. |
| `pollImage` | Switch | This control can be used to manually start and stop using your CPU to create snapshots from a RTSP source. If you have a snapshot URL setup in the binding, only then can this control can be used to update the Image channel. |
| `zoom` | Dimmer | Works with ONVIF cameras that can be moved. |

> | `acceptedCardNumber` | String (read only) | This channel shows the last Pass Card Number, authorized by Dahua VTO to open door. This could be used by Openhab's presence detection and security monitoring. Cards could be programmed into VTO without VTH with a help of Dahua SmartPSS soft <https://www.youtube.com/watch?v=qzR_ILT7j-0>. After that cards appear in VTO's web interface. The channel doesn't show rejected/unauthorised cards. |

Reviewer: ```suggestion
| `acceptedCardNumber` | String (read only) | This channel shows the last accepted access card number that opened the door. The channel doesn't show rejected/unauthorized cards. |
```
1. You spelled openHAB wrongly as we are picky about the capitals in our docs.
2. The channel will get re-used by other brands most likely in the future so removing mention of brands and keeping this short I think is a better idea. 
3. I probably would prefer to call the channel `lastAcceptedCard` as I would have to check how other brands that have a similar feature work. I suspect some may use a card hash ID, some may use a users name, and some may use a index number.

| `doorUnlock` | Switch | This channel could reflect Dahua VTO door lock state and at the same time send commands to door lock. Note that under some conditions Dahua doesn't send "lock off" message, so it's better to add expiration timer to corresponding item. |
| `doorContact` | Switch (read only) | Reflects Dahua VTO door open/closed contact state.

> | `exitButton` | Switch (read only) | Reflects Dahua VTO exit button state. This could be used to check for exit button's long clicks/double clicks, so the button could control other gates connected to Openhab, or outdoor lights.

Reviewer: openHAB not Openhab

| `whiteLED`                  | Dimmer | RW | Turn the visible white LED ON or OFF and if supported dim from 0-100%. |
| `zoom`                       | Dimmer | RW | Works with ONVIF cameras that can be moved. |
| `acceptedCardNumber` | String | R | This channel shows the last accepted access card number that opened the door. The channel doesn't show rejected/unauthorized cards. |
| `unacceptedCardNumber` | String | R | This channel shows the last unaccepted access card number that was read. |
| `doorUnlock` | Switch | RW | This channel could reflect door lock state and at the same time send commands to door lock. Note that under some conditions doorphone doesn't send "lock off" message, so it's better to add expiration timer to corresponding item. |
| `doorContact` | Switch | R | Reflects door open/closed contact state. |
| `exitButton` | Switch | R | Reflects exit button state. This could be used to check for exit button's long clicks/double clicks, so the button could control other gates connected to openHAB, or outdoor lights. |

> | `exitButtonEnable` | Switch | RW | This channel could be used to disable the exit button to provide additional security at night or when noone is home. |

Reviewer 2: In case you believe exitButtonEnable is a more appropriate name, then you have to use it with this name in other files.

> | `exitButtonEnable` | Switch | RW | This channel could be used to disable the exit button to provide additional security at night or when noone is home. |

Reviewer 3: Done
